src/main/java/com/google/api/codegen/transformer/OutputTransformer.java

```diff
@@ -207,8 +200,48 @@ public void run() {
     TypeModel type;
     if (baseIdentifier.equals(RESPONSE_PLACEHOLDER)) {
       view.variable(context.getNamer().getSampleResponseVarName(context));
+      boolean pageStreaming = context.getMethodConfig().getPageStreaming() != null;
+      boolean pageStreamingAndUseResourceName =
+          pageStreaming
```

Comment:
pageStreaming && context....?

Reply:
lol. I had a brain fart and was doing pageStreaming & context... and always got a npe. Now I understand why that kept happening :/